anyway/secrets.py

```diff
@@ -12,7 +12,17 @@ def get(secret_name: str) -> str:
         with open(secrets_dir + secret_name) as secret_file:
             return secret_file.read()
     logging.info("No secret dir found.")
-    return os.environ[secret_name]
+    return os.environ.get(secret_name)
+
+
+def get_with_default(secret_name: str, value: str) -> str:
+    logging.info("Reading secret for " + repr(secret_name))
+    secrets_dir = "/run/secrets/"
+    if os.path.isdir(secrets_dir):
+        with open(secrets_dir + secret_name) as secret_file:
+            return secret_file.read()
+    logging.info("No secret dir found, returning default: " + value)
+    return value
```

Comment:
I don't think we need this function - when we have a secret, there shouldn't be any useful default.

Also, if we need the function, then `get` should use it, and assert on the result.